src/test/scala/com/microsoft/hyperspace/index/RefreshIndexDeleteTests.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
```

Comment:
Thanks @rapoth for the suggestion. , NOTE TO REVIEWERS: This file is picked from a dependent pr: #142 And it doesn't need review here.